index.html

```diff
@@ -1,36 +1,233 @@
+        <span class="punto"></span>
+        <span>2015</span>
+      </div>
+      <p class="description"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+      <div class="pr-languages">
+        <div class="pr-lang"><span>html</span></div>
+        <div class="pr-lang"><span>css</span></div>
+        <div class="pr-lang"><span>javaScript</span></div>
+      </div>
+      </div>
+      <div class="project-button">
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+  <!--END OF WORK 2-->
+  <!--WORK 3-->
+  <div class="card">
+    <img class="card-picture" src="/images/picture3.PNG" alt="work example" />  
+    <div class="card-content">
+    <h2 class="title"> Tonic</h2>
+      <div class="info">
+        <span>CANOPY</span>
+        <span class="punto"></span>
+        <span>Back End Dev</span>
+        <span class="punto"></span>
+        <span>2015</span>
+      </div>
+      <p class="description"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+      <div class="pr-languages">
+        <div class="pr-lang"><span>html</span></div>
+        <div class="pr-lang"><span>css</span></div>
+        <div class="pr-lang"><span>javaScript</span></div>
+      </div>
+      </div>
+      <div class="project-button">
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+  <!--END OF WORK 3-->
+  <!--WORK 4-->
+  <div class="card">
+    <img class="card-picture" src="/images/picture4.PNG" alt="work example" />  
+    <div class="card-content">
+    <h2 class="title">Multi-Post Stories</h2>
+      <div class="info">
+        <span>CANOPY</span>
+        <span class="punto"></span>
+        <span>Back End Dev</span>
+        <span class="punto"></span>
+        <span>2015</span>
+      </div>
+      <p class="description"> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+      <div class="pr-languages">
+        <div class="pr-lang"><span>html</span></div>
+        <div class="pr-lang"><span>css</span></div>
+        <div class="pr-lang"><span>javaScript</span></div>
+      </div>
+      </div>
+      <div class="project-button">
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+  </div>
+<!--END OF WORK 4-->
+<!--END OF WORK SECTIONS-->
+<!--ABOUT ME SECTION-->
+<section id="about-me">
+    <div class="contenedor">
+      <div class="about-me-content">
+      <h3 class="title"> About Myself</h3>
+      <p class="description"> I'm a software developer! I can help you build a product ,feature or website. Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <p class="connect">LETS CONNECT</p>
+        <div class="social-network">
+            <img src="images/twitter-logo.svg" alt="twitter" />
```

Comment:
- [ ] Nicely done, following the requirement kindly list the social icons using `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag)